core/src/main/resources/operatorsScore-dataproc-gke-t4.csv

```diff
@@ -0,0 +1,268 @@
+DateSub,3.65
+DayOfMonth,3.65
+DayOfWeek,3.65
+DayOfYear,3.65
+DenseRank,3.65
+Divide,3.65
+DynamicPruningExpression,3.65
+ElementAt,3.65
+EndsWith,3.65
+EqualNullSafe,3.65
+EqualTo,3.65
+Exp,3.65
+Explode,3.65
+Expm1,3.65
+First,3.65
+Flatten,3.65
+Floor,3.65
+FromUTCTimestamp,3.65
+FromUnixTime,3.65
+GetArrayItem,3.65
+GetArrayStructFields,3.65
+GetJsonObject,3.65
+GetMapValue,3.65
+GetStructField,3.65
+GetTimestamp,3.65
+GreaterThan,3.65
+GreaterThanOrEqual,3.65
+Greatest,3.65
+HiveGenericUDF,3.65
+HiveSimpleUDF,3.65
+Hour,3.65
+Hypot,3.65
+If,3.65
+In,3.65
+InSet,3.65
+InitCap,3.65
+InputFileBlockLength,3.65
+InputFileBlockStart,3.65
+InputFileName,3.65
+IntegralDivide,3.65
+IsNaN,3.65
+IsNotNull,3.65
+IsNull,3.65
+JsonToStructs,3.65
+JsonTuple,3.65
+KnownFloatingPointNormalized,3.65
+KnownNotNull,3.65
+Lag,3.65
+LambdaFunction,3.65
+Last,3.65
+LastDay,3.65
+Lead,3.65
+Least,3.65
+Length,3.65
+LessThan,3.65
+LessThanOrEqual,3.65
+Like,3.65
+Literal,3.65
+Log,3.65
+Log10,3.65
+Log1p,3.65
+Log2,3.65
+Logarithm,3.65
+Lower,3.65
+MakeDecimal,3.65
+MapConcat,3.65
+MapEntries,3.65
+MapFilter,3.65
+MapKeys,3.65
+MapValues,3.65
+Max,3.65
+Md5,3.65
+MicrosToTimestamp,3.65
+MillisToTimestamp,3.65
+Min,3.65
+Minute,3.65
+MonotonicallyIncreasingID,3.65
+Month,3.65
+Multiply,3.65
+Murmur3Hash,3.65
+NaNvl,3.65
+NamedLambdaVariable,3.65
+NormalizeNaNAndZero,3.65
+Not,3.65
+NthValue,3.65
+OctetLength,3.65
+Or,3.65
+PercentRank,3.65
+PivotFirst,3.65
+Pmod,3.65
+PosExplode,3.65
+Pow,3.65
+PreciseTimestampConversion,3.65
+PromotePrecision,3.65
+PythonUDF,3.65
+Quarter,3.65
+RLike,3.65
+RaiseError,3.65
+Rand,3.65
+Rank,3.65
+RegExpExtract,3.65
+RegExpExtractAll,3.65
+RegExpReplace,3.65
+Remainder,3.65
+ReplicateRows,3.65
+Reverse,3.65
+Rint,3.65
+Round,3.65
+RowNumber,3.65
+ScalaUDF,3.65
+ScalarSubquery,3.65
+Second,3.65
+SecondsToTimestamp,3.65
+Sequence,3.65
+ShiftLeft,3.65
+ShiftRight,3.65
+ShiftRightUnsigned,3.65
+Signum,3.65
+Sin,3.65
+Sinh,3.65
+Size,3.65
+SortArray,3.65
+SortOrder,3.65
+SparkPartitionID,3.65
+SpecifiedWindowFrame,3.65
+Sqrt,3.65
+StartsWith,3.65
+StddevPop,3.65
+StddevSamp,3.65
+StringInstr,3.65
+StringLPad,3.65
+StringLocate,3.65
+StringRPad,3.65
+StringRepeat,3.65
+StringReplace,3.65
+StringSplit,3.65
+StringToMap,3.65
+StringTranslate,3.65
+StringTrim,3.65
+StringTrimLeft,3.65
+StringTrimRight,3.65
+Substring,3.65
+SubstringIndex,3.65
+Subtract,3.65
+Sum,3.65
+Tan,3.65
+Tanh,3.65
+TimeAdd,3.65
+ToDegrees,3.65
+ToRadians,3.65
+ToUnixTimestamp,3.65
+TransformKeys,3.65
+TransformValues,3.65
+UnaryMinus,3.65
+UnaryPositive,3.65
+UnboundedFollowing$,3.65
+UnboundedPreceding$,3.65
+UnixTimestamp,3.65
+UnscaledValue,3.65
+Upper,3.65
+VariancePop,3.65
+VarianceSamp,3.65
+WeekDay,3.65
+WindowExpression,3.65
+WindowSpecDefinition,3.65
+XxHash64,3.65
+Year,3.65
+AggregateInPandasExec,1.2
+ArrowEvalPythonExec,1.2
+FlatMapGroupsInPandasExec,1.2
+FlatMapCoGroupsInPandasExec,1.2
+MapInPandasExec,1.2
+WindowInPandasExec,1.2
+KMeans-pyspark,8.86
```

Comment:
Operators from KMeans-pyspark to XBoost-scala s/b removed as those are Spark-ML functions.

Reply:
Thanks @nartal1. Removed them.